dev/src/dev.clj

```diff
@@ -212,7 +212,7 @@
   first arg:
 
     (dev/query-jdbc-db
-     [:sqlserver 'test-data-with-time]
+     [:sqlserver 'time-test-data]
```

Comment:
I renamed `test-data-with-timezones` -> `timezone-test-data` to keep tables under 30 characters.
so I renamed `test-data-with-time` -> `time-test-data` for consistency